sdk/examples/example.ts

```diff
@@ -2,61 +2,86 @@
 // Modifications Copyright (c) 2025 IOTA Stiftung
 // SPDX-License-Identifier: Apache-2.0
 
-import { getFullnodeUrl, IotaClient } from '@iota/iota-sdk/client';
+import { getNetwork, IotaClient } from '@iota/iota-sdk/client';
+import { requestIotaFromFaucetV0 } from '@iota/iota-sdk/faucet';
+import { Ed25519Keypair } from '@iota/iota-sdk/keypairs/ed25519';
 import { Transaction } from '@iota/iota-sdk/transactions';
 
 import { IotaNamesClient } from '../src/iota-names-client.js';
 import { IotaNamesTransaction } from '../src/iota-names-transaction.js';
 
-// Initialize and execute the IotaNamesClient to fetch the renewal price list
 (async () => {
-    const network = 'testnet';
-    // Step 1: Create a IotaClient instance
+    const domain = `cool-domain-${(Math.random() * 10000000).toString().split('.').join('')}.iota`;
```

Comment:
```suggestion
    const domain = `cool-domain-${Math.floor(Math.random() * 10000000)}.iota`;
```

Not actually sure why the number is there but this is maybe slightly better.

Reply:
Lol, true, the number is kinda useless in my code, I didnt think it much tbh